taskcluster/ci/ui-test/kind.yml

```
@@ -49,7 +49,7 @@ jobs:
     x86-debug:
         description: Test Fenix
         run-on-tasks-for: [github-pull-request, github-push]
-        run-on-git-branches: [master]
+        run-on-git-branches: ["^((?!releases_)(?!releases/).)*$"]
```

Great idea on filtering out the release branches 👍 I hadn't thought of it 🙂

That made me realize we can simplify the regexes below and this one too. I think that one is equivalent: 

```suggestion
        run-on-git-branches: ["^((?!releases[_/]).+)$"]
```